app/controllers/packages_controller.rb

```diff
@@ -18,6 +18,26 @@ def index
            meta: { totalResults: @packages.totalResults }
   end
 
+  def create
+    package_validation = Validation::CustomPackageParameters.new(package_params)
+
+    if package_validation.valid?
+      package_response = packages.create rm_api_package
+
+      # Instead of trying to merge objects together, lets just request
+      # the newly created package from RM API and render that.
```

Comment:
👍 gooood move

Reply:
although what if it doesn't exist?

Reply:
That's a good question. I could totally see an issue where we request the newly created package and the API doesn't return it. Worst case the package is still created and you get an empty response (instead of a full package)